deploy/helm/sumologic/values.yaml

```diff
@@ -4602,7 +4604,7 @@ otellogs:
   ## Configure image for Opentelemetry Collector
   image:
     repository: public.ecr.aws/sumologic/sumologic-otel-collector
-    tag: 0.47.0-sumo-0
+    tag: v0.49.0-sumo-0
```

Comment:
FYI, this is now available: https://github.com/SumoLogic/sumologic-otel-collector/releases/tag/v0.49.0-sumo-0.

Reply:
updated